podcasts/Onboarding/Plus/UpgradeLandingView.swift

```diff
@@ -159,7 +158,7 @@ extension UpgradeTier {
     }
 
     static var patron: UpgradeTier {
-        UpgradeTier(tier: .patron, iconName: "patron-heart", title: "Patron", yearlyIdentifier: "com.pocketcasts.patron.yearly", monthlyIdentifier: "com.pocketcasts.patron.monthly", description: L10n.patronDescription, buttonLabel: L10n.patronSubscribeTo, buttonColor: Color(hex: "6046F5"), buttonForegroundColor: .white, features: [
+        UpgradeTier(tier: .patron, iconName: "patron-heart", title: "Patron", plan: .patron, description: L10n.patronDescription, buttonLabel: L10n.patronSubscribeTo, buttonColor: Color(hex: "6046F5"), buttonForegroundColor: .white, features: [
             TierFeature(iconName: "patron-everything", title: "Everything in Plus"),
```

Comment:
Should we add Everything in Plus to the localized strings?

Reply:
Ops. Done in a3b35698bfbd9c9c3b4b6b70d4e972a4611ac9bb